cmd/activator/main.go

```diff
@@ -50,6 +49,7 @@ import (
 const (
 	maxUploadBytes = 32e6 // 32MB - same as app engine
 	logLevelKey    = "activator"
+	component      = "activator"
```

Comment:
Do we need both of these keys?  Perhaps just generalize `logLevelKey`? :)

Reply:
That's a good point. I think it would be better to clean up after this pr is in, since we have logLevelKey in other files like cmd/webhook/main.go and cmd/controller/main.go. If you agree, I'll open an issue and assign to me.

Reply:
Sure.

Reply:
https://github.com/knative/serving/issues/2370